Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM maven:3.5-jdk-8
```

Comment:
I'd like to avoid docker dependencies if possible, can you strip these bits out?